### PR TITLE
mail/rspamd: fix permissions due to upstream changes

### PR DESCRIPTION
**Pull request by mimugmail**

### CODE REVIEW

File: mail/rspamd/src/opnsense/scripts/rspamd/setup.sh [hunk 1]
```diff
@@ -7,8 +7,8 @@ mkdir -p /var/run/rspamd
 chmod +r /usr/local/etc/rspamd/local.d/*
 chmod +r /usr/local/etc/rspamd/maps.d/*
 chmod o+rx /usr/local/etc/rspamd/local.d
-chown -R nobody /var/log/rspamd
+chown -R rspamd /var/log/rspamd
```
Comment (fichtner) [inline mail/rspamd/src/opnsense/scripts/rspamd/setup.sh]: isn't this in there twice?

Comment (mimugmail) [reply mail/rspamd/src/opnsense/scripts/rspamd/setup.sh]: Indeed

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by fabianfrz** [comment]
@fichtner no problem.

**Pull request merged**